### ISSUE TITLE
Error occurred while following through dialog box: 'follow'

### ISSUE THREAD

**Issue by hasangokdag** [InstaPy/InstaPy#4639]
When trying to follow a user's followers i get the following error:
```
Error occurred while following through dialog box:
'follow'
```

This happens at the stage where it opens up followers modal box and scrolls down on the list.
After grabbing the follower list it starts following 2-3 people then

```
[5764:8256:0709/092815.040:ERROR:broker_win.cc(55)] Error reading broker pipe: The pipe has been ended. (0x6D)
[5764:8256:0709/092815.040:ERROR:broker_win.cc(137)] Error sending sync broker message: The pipe is being closed. (0xE8)
[5764:8256:0709/092815.040:ERROR:command_buffer_proxy_impl.cc(94)] ContextResult::kFatalFailure: AllocateAndMapSharedMemory failed
```
and the session stops.

After session stops i get the following traceback
```
Traceback (most recent call last):
  File ".\mecmua3.py", line 112, in <module>
    interact=True)
  File "C:\Users\gokdagh\AppData\Local\Programs\Python\Python37-32\lib\site-packages\instapy\instapy.py", line 3255, in follow_user_followers
    interact)
  File "C:\Users\gokdagh\AppData\Local\Programs\Python\Python37-32\lib\site-packages\instapy\instapy.py", line 1088, in follow_by_list
    self.logfolder)
  File "C:\Users\gokdagh\AppData\Local\Programs\Python\Python37-32\lib\site-packages\instapy\unfollow_util.py", line 764, in follow_user
    naply = get_action_delay("follow")
  File "C:\Users\gokdagh\AppData\Local\Programs\Python\Python37-32\lib\site-packages\instapy\util.py", line 1771, in get_action_delay
    custom_delay = config[action]
KeyError: 'follow'
```

Running it on Win10 with python 3.7.3

**Comment by hasangokdag** [comment]
I turned off the simulation,
`session.set_simulation(enabled=False)`

Now this only occurs after following the first user
`[11232:3284:0709/094019.835:ERROR:broker_win.cc(55)] Error reading broker pipe: The pipe has been ended. (0x6D)`
with the exact same traceback after session end

**Comment by GitNees** [comment]
Try liking something, i get same problem with likes and crashes with same errors you have.

**Comment by GoldenPolo** [comment]
Same for me on MacOS :

File "/Library/Python/2.7/site-packages/instapy/instapy.py", line 3255, in follow_user_followers interact) 
File "/Library/Python/2.7/site-packages/instapy/instapy.py", line 1088, in follow_by_list self.logfolder) 
File "/Library/Python/2.7/site-packages/instapy/unfollow_util.py", line 764, in follow_user naply = get_action_delay("follow") 
File "/Library/Python/2.7/site-packages/instapy/util.py", line 1771, in get_action_delay custom_delay = config[action] 
KeyError: 'follow'


Any ideas to fix it ?

**Comment by mrnfrancesco** [comment]
Same issue of #4610 
Already fixed by #4640

**Comment by breuerfelix** [comment]
update to the latest instapy version to get the fix :)
